### PR TITLE
startTime, imageQuality ++

### PR DESCRIPTION
**Pull request by magnurud**

### CODE REVIEW

File: las/client.py [hunk 1]
```diff
@@ -459,8 +464,11 @@ def create_prediction(
         :param max_pages: Maximum number of pages to run predictions on
         :type max_pages: Optional[int]
         :param auto_rotate: Whether or not to let the API try different rotations on\
- the document when running predictions
+            the document when running predictions
         :type auto_rotate: Optional[bool]
+        :param image_quality: image quality for prediction "low|medium|high". \
```
Comment (augustak) [inline las/client.py:469]: LOW|HIGH

File: las/client.py [hunk 2]
```diff
@@ -37,6 +37,10 @@ def _json_decode(response):
         response.raise_for_status()
         return response.json()
     except JSONDecodeError as e:
+
+        if response.status_code == 204:
+            return {'Your requested executed successfully': '204'}
```
Comment (hexjelly) [inline las/client.py:42]:
```suggestion
            return {'Your request executed successfully': '204'}
```

File: las/client.py [hunk 3]
```diff
@@ -847,9 +857,11 @@ def update_transition_execution(
         :param status: Status of the execution 'succeeded|failed'
         :type status: str
         :param output: Output from the execution, required when status is 'succeded'
-        :type output: Optional[str]
+        :type output: Optional[dict]
         :param error: Error from the execution, required when status is 'failed', needs to contain 'message'
-        :type error: Optional[str]
+        :type error: Optional[dict]
+        :param start_time: Utc start time that will replace the original start time of the execution
+        :type start_time: Optional[str]
```
Comment (augustak) [inline las/client.py:864]: Should we accept datetime aswell?

File: las/client.py [hunk 4]
```diff
@@ -861,9 +874,31 @@ def update_transition_execution(
             'status': status,
             'output': output,
             'error': error,
+            'startTime': f'{start_time}' if start_time else start_time,
         }
         return self._make_request(requests.patch, url, body=dictstrip(body))
 
+    def send_heartbeat(self, transition_id: str, execution_id: str) -> Dict:
+        """Send heartbeat for a manual execution,
```
Comment (augustak) [inline las/client.py:882]: "Send heartbeat for a manual execution to signal that we are still working on it. Must be done at minimum once every 60 seconds or the transition execution will time out"

File: las/client.py [hunk 5]
```diff
@@ -861,9 +874,32 @@ def update_transition_execution(
             'status': status,
             'output': output,
             'error': error,
+            'startTime': f'{start_time}' if start_time else start_time,
```
Comment (augustak) [inline las/client.py:877]: Would else None be more readable?
```suggestion
            'startTime': f'{start_time}' if start_time else None,
```

File: las/client.py [hunk 6]
```diff
@@ -861,9 +874,32 @@ def update_transition_execution(
...
+        :param execution_id: Id of the transition execution
+        :type execution_id: str
+        :return: Empty response
+        :rtype: None
+
+        :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
+ :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
+        """
+        endpoint = f'/transitions/{transition_id}/executions/{execution_id}/heartbeats'
+        return self._make_request(requests.post, endpoint, body={})
```
Comment (augustak) [inline las/client.py:901]: Do we need body={}?

Comment (magnurud) [reply las/client.py:901]: yes

**Review by augustak — APPROVED** [review]

**Pull request merged**